website/content/docs/jekyll.md

```diff
@@ -0,0 +1,463 @@
+
+- The `label` property is the string used to identify the collection in the cms UI. We'll set it to `"Blog"`
+- `folder` should be set to the directory containing the files we want the cms to be able to create and edit. We'll start with `"_posts/"`
+- The `fields` property requires an array of field objects. We'll start with a collection with only the property `name` defined. Setting the `fields` property will take care of the `files` property error as well.
+
+With these update, `config.yml` should look like this.
+
+```yml
+# config.yml
+
+backend:
+  name: test-repo
+media_folder: 'assets/uploads'
+collections:
+  - name: 'blog'
+    label: 'Blog'
+    folder: '_posts/'
+    fields:
+      - { name: Title }
+```
+
+Back in the browser you should see.
+
+![Netlify CMS signup screenshot](https://www.netlifycms.org/img/screenshot-test-repo-login.png?raw=true)
+
+Log in and you should see.
+
+![empty blog collection screenshot](https://www.netlifycms.org/img/screenshot-empty-blog-collection.png?raw=true)
+
+Awesome, the CMS is running without errors! Great, but you might have noticed the CMS isn't displaying data from the three markdown files in the `_posts/` directory. The reason is `test-repo` uses local browser storage and doesn't have access to your file system. In fact, none of the Netlify CMS backends can interact with local git repositories. This is a common point of confusion and bears repeating: NETLIFY CMS CANNOT INTERACT WITH LOCAL GIT REPOSITORIES.
+
+Next, we'll setup the Backend and Authentication so you can start updating content on your Jekyll site.
+
+### Backend and Authentication
+
+You have [lots of options](https://www.netlifycms.org/docs/authentication-backends/) for giving Netlify CMS permission to push commits to the git repository for your Jekyll project. In this guide - we're using Netlify's Identity service.
+
+Replace the `test-repo` backend configuration `git-gateway`
+
+```yml
+//
+# config.yml
+backend:
+  name: git-gateway
+  branch: master # Branch to update (optional; defaults to master)
+```
+
+In your browser, go to `https://app.netlify.com/sites/{YOUR_PROJECT_NAME}/settings/identity`
+
+Click `Enable Identity`
+
+Scroll down to the **Registration** section and add external OAuth providers (_optional_)
+
+Scroll down to the **Services** section and click `Enable Git Gateway`
+
+Add the `netlify-identity-widget.js` script to the head of `admin/index.html` and `_layouts/default.html`
+
+```html
+<script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
+```
+
+Add the following redirect script to the bottom of the body in `_layouts/default.html`
+
+```html
+<script>
+  if (window.netlifyIdentity) {
+    window.netlifyIdentity.on("init", user => {
+      if (!user) {
+        window.netlifyIdentity.on("login", () => {
+          document.location.href = "/admin/";
+        });
+      }
+    });
+  }
+</script>
+```
```

Comment:
The detailed walkthrough of setting up identity should be replaced with a link to the git-gateway page in these docs - otherwise if the details of how this works changes, we have multiple places to update, and likely won't catch them all. Better to have a single source of truth and just link there.

Reply:
I removed setting backend to test-repo entirely and just added a links for setting up Identity/Git-gateway and adding the Identity widget